src/Bicep.Core.UnitTests/Syntax/ImportSpecificationTest.cs

```diff
@@ -9,22 +9,21 @@
 [TestClass]
 public class ImportSpecificationTests
 {
-    [DataTestMethod]
+    [TestMethod]
```

Comment:
nit: Should use DataTestMethod instead? Also, it seems the test class is not in a namespace.

Reply:
Either works, but there's a bug in the VSCode C# extension where it no longer detects `DataTestMethod` - this is me being lazy because I wanted to run this test 😄